### PR TITLE
Hotfixes/UI

### PR DESCRIPTION
**Pull request by silvap-javier**
# Hotfixes/UI

## Issues

closes #

## Dependencies

#

## Changes : Resolved

- After introducing mnemonic to import, Import wallet button should only be clickable once. If not it imports the same account multiple times.
- At Security settings, delete wallet. After authorising delete, are you sure? modal doesn’t close.
- After delete data, are you sure? modal doesn’t close.
- At Succesful transaction screen. Confirm text doesn’t show up.
- Change theme switch doesn’t respond and won’t change theme.
- En la wallet card, al texto de los botones les falta un poco de espacio y el texto sale negro, debería ser blanco
- Al "No hay NFTs" le falta espacio

### CODE REVIEW

File: src/module/sdk/core/transaction.service.ts [hunk 1]
```diff
@@ -499,7 +499,7 @@ export class TransactionService {
             throw new Error("Insufficient capacity");
         }
         if (changeAmount > 0 && changeCapacity < helpers.minimalCellCapacityCompatible(changeCell).toBigInt()) {
-            throw new Error("Insufficient capacity for change cell");
+            throw "insufficient_capacity_for_change_cell";
```
[the remark this answers is not in the document]
Comment (silvap-javier) [reply src/module/sdk/core/transaction.service.ts]: Done

File: src/module/common/component/input/LabeledIconButton/LabeledIconButton.tsx [hunk 2]
```diff
@@ -3,14 +3,14 @@ import IconButton from "../IconButton/IconButton";
...
-export default function LabeledIconButton({ label, labelColor, onPress, ...iconButtonProps }: LabeledIconButtonProps): JSX.Element {
+export default function LabeledIconButton({ label, onPress, ...iconButtonProps }: LabeledIconButtonProps): JSX.Element {
     return (
         <TouchableWithoutFeedback onPress={onPress}>
-            <Col alignItems="center">
+            <Col alignItems="center" gap={3}>
                 <View style={{ marginHorizontal: "auto" }}>
                     <IconButton onPress={onPress} {...iconButtonProps} />
                 </View>
-                <Typography variant="body4Regular" color={labelColor} adjustsFontSizeToFit>
```
[the remark this answers is not in the document]
Comment (silvap-javier) [reply src/module/common/component/input/LabeledIconButton/LabeledIconButton.tsx:13]: import { useTheme } from "@peersyst/react-native-components";
import LabeledIconButton from "../LabeledIconButton/LabeledIconButton";
import { ActionIconButtonProps } from "./ActionIconButton.types";

export default function ActionIconButton({ isActive, action, ...rest }: ActionIconButtonProps): JSX.Element {
    const {
        palette: { gray, overlay },
    } = useTheme();
    return (
        <LabeledIconButton
            labelColor={isActive ? gray[0] : overlay[100]["48%"]}
            variant={isActive ? "secondary" : "outlined"}
            onPress={action}
            {...rest}
        />
    );
    
    Solo se usa aqui, por eso creo que no es necesario

File: src/query/handleErrorMessage.ts [hunk 3]
```diff
@@ -12,7 +12,10 @@ export type UseHandleErrorMessage = (error: ApiError | any) => HandleApiErrorMes
 export function handleErrorMessage(error: ApiError | any, translate: typeof i18n.t): HandleApiErrorMessageResult {
     const code = error.body?.statusCode || error.status;
     const message = error.body?.message || error.statusText;
-    if (!code || code === 500) return { message: translate("somethingWentWrong", { ns: "error" }), type: "error" };
+
+    if (error && !code && !message) {
+        return { message: translate(error in en.error ? error : "somethingWentWrong", { ns: "error" }), type: "error" };
```
[the remark this answers is not in the document]
Comment (silvap-javier) [reply src/query/handleErrorMessage.ts]: Done

File: src/module/common/component/input/LabeledIconButton/LabeledIconButton.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (JordiParraCrespo) [reply src/module/common/component/input/LabeledIconButton/LabeledIconButton.tsx:13]: Habria que quitar la prop entonces no?

[the remark this answers is not in the document]
Comment (silvap-javier) [reply src/module/common/component/input/LabeledIconButton/LabeledIconButton.tsx:13]: Done

**Review by AgustinMJ — APPROVED** [review]

**Pull request merged**